src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
The class should have an explicit access modifier to define its visibility clearly. In this case, it seems like it should be public.